tests/data_validation/data_validation_list_component.test.ts

```diff
@@ -233,6 +238,32 @@ describe("autocomplete in composer", () => {
       expect(getCellContent(model, "A1")).toBe("hello");
       expect(model.getters.getEditionMode()).toBe("inactive");
     });
+
+    test("Dropdown lists are kept when clearing the content of a non-empty cell", () => {
+      expect(model.getters.getDataValidationListCellsPositions()).toEqual([
+        { sheetId: model.getters.getActiveSheetId(), col: 0, row: 0 },
+      ]);
+      setCellContent(model, "A1", "hello");
+      deleteContent(model, ["A1"]);
+      expect(model.getters.getDataValidationListCellsPositions()).toHaveLength(1);
+    });
+
+    test("Dropdown lists are removed when clearing the content of an empty cell", () => {
+      expect(model.getters.getDataValidationListCellsPositions()).toEqual([
+        { sheetId: model.getters.getActiveSheetId(), col: 0, row: 0 },
+      ]);
+      setCellContent(model, "A1", "");
+      deleteContent(model, ["A1"]);
+      expect(model.getters.getDataValidationListCellsPositions()).toHaveLength(0);
+    });
+
+    test("Dropdown lists are removed when clearing the content of an empty cell", () => {
```

Comment:
this test title is the same as the test above. So I'm not sure what it's supposed to test